md/applicationCreation.md

```diff
@@ -0,0 +1,69 @@
+:::
+
+From Bonita BPM Studio, click on **Development** -> **Application Descriptor** -> **New...**  
+It will create an empty application file with a default name and open an editor. You can **rename** this Application file from the graphical UI or from the menu **Open**.  
+
+![Empty Application File](images/applicationDescriptors/emptyApplicationFile.png)  
+Here is the application descriptor graphical UI provided. It is bound to the XML source, so you can use it to build your application descriptor without writing xml.  
+You can switch from the graphical UI to the xml source at any moment, using the tabs _Editor_ and _Source_ at the bottom of the editor.  
+A toolbar is embedded to the editor, use it to:  
+
+ - **Add** an application descriptor to this application file
+ - **Deploy** this application file (i.e all the application descriptors in this file).⚠  For _development_ purposes, it also deploys their referenced pages and layout, but if those resources use REST API Extensions, you have to deploy those manually. In _production_ environment, you still need to deploy pages, layouts and REST API Extensions manually **before** to deploy the application descriptor.   
+ - **Rename** this application file
+ - **Export** this application file (download the .xml)
+ - **Delete** this application file from your current workspace
+
+![Application Descriptor Editor](images/applicationDescriptors/applicationDescriptorEditor.png)
+
+The application **Token** is the identifier of your Application Descriptor; it has to be unique. It is used to define the URL root of your Application:  _../apps/APPLICATIONTOKEN/.._  
+
+the fields _Profile_, _Layout_, _Theme_, _Application Page_ and _Home page token_ come with an auto complete feature.  
+Be aware that for the moment, we can only suggest choices known by the Studio and the UI designer.  
+It means that custom profiles, custom themes or custom pages / layouts only known by the portal (not created with the UI Designer) won't be suggested. You can still reference an unknown resource, but you will have to write the exact identifier by yourself, and make sure the resource is imported onto the portal **before** you deploy/import the application descriptor.  
+
+There are two ways to reference pages used by your Application. In the _Navigation_ table or in the _Orphan pages_ table.  
+Use the _Navigation_ table to create the application menus/submenus, and bind those menus to pages.  
+A page needs a unique token (unique in the context of the application descriptor). It will be use to define the URL to reach this page: _../apps/applicationToken/PAGETOKEN_  
+Use the Orphan pages table to reference pages that will be accessed using the associated URL, but won't be attached to a menu. It can be the application home page, or pages accessible from other pages but not from a menu.  
```

Comment:
forgot italic for Orphan pages => Use the _Orphan pages_ table to reference pages that will be accessed using the associated URL, but won't be attached to a menu.